azure-spring-boot-parent/pom.xml

```diff
@@ -65,6 +66,26 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk9+</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>${java.version}</maven.compiler.release>
```

Comment:
The "release" flag supersedes the "target" and "source" flags in Javac and the maven-compiler-plugin since JDK9.
It additionally enables JDK9+ to compile in a "compatibility-mode" to Java 8.